config/initializers/csp.rb

```diff
@@ -1,11 +1,11 @@
-SecureHeaders::Configuration.default do |config|  
+SecureHeaders::Configuration.default do |config|
   config.csp = {
     preserve_schemes: true, # default: false.
     default_src: %w('self'),
-    script_src: %w('self' 'unsafe-eval'),
+    script_src: %w('self' 'unsafe-eval' 'unsafe-inline'),
```

Comment:
I'm not familiar with CSP or its workarounds. Is there a doc or should I let you and darth handle it?